sopel/cli/plugins.py

```diff
@@ -149,7 +153,12 @@ def build_parser():
 
 
 def handle_list(options):
-    """List Sopel plugins"""
+    """List Sopel plugins
+
+    :param options: parsed arguments
+    :type options: ``argparse.Namespace``
+    :return: Return 0 if everything went fine.
```

Comment:
```suggestion
    :return: 0 if everything went fine
```

Here again, weird that 0 is the only possible return value (exit code).